public/components/app.tsx

```diff
@@ -80,7 +86,7 @@ export const App = ({ CoreStart, pplService }: ObservabilityAppDeps) => {
                     parentBreadcrumb,
                     {
                       text: 'Event analytics',
-                      href: '#/explorer/home',
+                      href: '/explorer/events',
```

Comment:
Was this an intended change? shouldn't this line remain '#/explorer/home'

Reply:
yes, this was intended. eric changed the path for event explorer in a previous commit